plugins/modules/remote_management/redfish/ilo_redfish_command.py

```diff
@@ -0,0 +1,343 @@
+            res = rf_utils._find_systems_resource()
+            if res['ret'] is False:
+                result[command] = res
+                module.fail_json(msg=to_native(result))
+
+            if command == "VerifyServiceBiosAttributes":
+                if not module.params.get("service_attributes") and module.params.get("service_attributes") != {}:
+                    result[command]['ret'] = False
+                    result[command]['msg'] = "service_attributes params is required"
+                    module.fail_json(result)
+                result[command] = rf_utils.verify_service_bios_attributes(module.params["service_attributes"])
+            elif command == "VerifyBiosAttributes":
+                if not module.params.get("bios_attributes") and module.params.get("bios_attributes") != {}:
+                    result[command]['ret'] = False
+                    result[command]['msg'] = "bios_attributes params is required"
+                    module.fail_json(result)
+                result[command] = rf_utils.verify_bios_attributes(module.params["bios_attributes"])
+            elif command == "VerifyLogicalDrives":
+                if not module.params.get("raid_details") and module.params.get("raid_details") != []:
+                    result[command]['ret'] = False
+                    result[command]['msg'] = "raid_details params is required"
+                    module.fail_json(result)
+                result[command] = rf_utils.verify_logical_drives(module.params["raid_details"], True)
+            elif command == "VerifySpecifiedLogicalDrives":
+                if not module.params.get("raid_details") and module.params.get("raid_details") != []:
+                    result[command]['ret'] = False
+                    result[command]['msg'] = "raid_details params is required"
+                    module.fail_json(result)
+                result[command] = rf_utils.verify_logical_drives(module.params["raid_details"], False)
+            elif command == "VerifyUefiBootOrder":
+                if not module.params["uefi_boot_order"]:
+                    result[command]['ret'] = False
+                    result[command]['msg'] = "uefi_boot_order params is required"
+                    module.fail_json(result)
+                result[command] = rf_utils.verify_uefi_boot_order(module.params["uefi_boot_order"])
+            elif command == "CheckRebootStatus":
+                result[command] = rf_utils.check_reboot_status()
+            elif command == "GetSpecifiedLogicalDrives":
+                if not module.params["logical_drives_names"]:
+                    result[command]['ret'] = False
+                    result[command]['msg'] = "logical_drives_names params is required"
+                    module.fail_json(result)
+                result[command] = rf_utils.get_specified_logical_drives(module.params["logical_drives_names"])
+
+    # Return data back or fail with proper message
+    if not result[command]['ret']:
```

Comment:
this is out of the loop from line 287, so it will only test the last command in the list